tests/causal_estimators/test_causalml_estimator.py

```diff
@@ -0,0 +1,216 @@
+import pytest
+import subprocess
+import sys
+
+from dowhy import CausalModel
+from dowhy.datasets import linear_dataset
+
+# To skip the tests if we cannot install causalml
+installed_failed = False
+# Hack to install causalml if not already present
+try:
+    __import__("causalml")

```

Comment:
why not the normal import statement?
"import causalml"
does this version give any benefits?

Reply:
We can always replace the string with a variable. In a nutshell, it is more extensible than manually importing dependencies.